smartcontractkit/chainlink
# Propagating evmChainID in job specs

Closes https://app.shortcut.com/chainlinklabs/story/17806/ui-job-definition-does-not-show-evmchainid

How to test:
- create a job with `evmChainID = 42` in the spec definition;
- after the job is created, open that job's spec and see `evmChainID = 42` is rendered.

`evmChainID` is supported for the following types:
- direct request
- flux monitor
- keeper
- VRF
- OCR
